### PR TITLE
Add handshake mapping

### PR DESCRIPTION
**Pull request by mjoyce91**
Can use with https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/529 to test `revoked` status

### CODE REVIEW

File: src/actions/bidList/bidList.js [hunk 1]
```diff
@@ -259,7 +260,8 @@ export function clientBidListFetchData(ordering = 'draft_date') {
       api().get(endpoint)
         .then((response) => {
           batch(() => {
-            dispatch(clientBidListFetchDataSuccess({ results: get(response, 'data.results', []) }));
+            console.log('x');
```
Comment (mjoyce91) [inline src/actions/bidList/bidList.js:263]: can remove

File: src/Components/BidTracker/BidHelpers.test.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ describe('shouldShowAlert function', () => {
     expect(result).toBe(true);
   });
 
-  it('returns false for an invalid status but when is_paneling_today === true', () => {
+  xit('returns false for an invalid status but when is_paneling_today === true', () => {
```
Comment (elizabeth-jimenez) [inline src/Components/BidTracker/BidHelpers.test.js:51]: 👀

File: src/Constants/BidData.js [hunk 3]
```diff
@@ -4,14 +4,19 @@ export const CLOSED_PROP = 'closed';
...
 export const SUBMITTED_PROP = 'submitted';
 export const PANEL_RESCHEDULED_PROP = 'panel_rescheduled';
 export const HAND_SHAKE_NEEDS_REGISTER_PROP = 'handshake_needs_registered';
 
+// tmap-specific props
+export const HAND_SHAKE_OFFERED_PROP = 'handshake_offered';
+export const HAND_SHAKE_OFFER_ACCEPTED_PROP = 'handshake_offer_accepted';
+export const HAND_SHAKE_OFFER_DECLINED_PROP = 'handshake_offer_declined';
+export const HAND_SHAKE_REVOKED_PROP = 'handshake_offer_revoked';
+
```
Comment (elizabeth-jimenez) [inline src/Constants/BidData.js:19]: should we add a 'not_offered' one here?

File: src/actions/bidList/helpers.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      // Use a new ACCEPTED status, clone the tracker status of HAND_SHAKE_OFFERED_PROP for
+      // this and use a prop to show the offer has already been accepted
+    }
+    // Handshake accepted, give CDO ability to register
+    if (m[hs] === HAND_SHAKE_OFFER_ACCEPTED_PROP && m[status] === HAND_SHAKE_NEEDS_REGISTER_PROP) {
+      override[status] = HAND_SHAKE_NEEDS_REGISTER_PROP;
+      // Use a new CAN_DECLINE_OR_REGISTER status,
+      // clone the tracker status of HAND_SHAKE_OFFERED_PROP for
+      // this and use a prop to show the offer has already been accepted, and allow the CDO to
+      // decline or register
```
Comment (elizabeth-jimenez) [inline src/actions/bidList/helpers.js:42]: 😍

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🍬 🍥 👍

**Pull request merged**